usaspending_api/broker/management/commands/fpds_nightly_loader.py

```diff
@@ -401,6 +393,21 @@ def load_specific_transactions(self, detached_award_procurement_ids):
 
         logger.info("FPDS SPECIFIC (RE)LOAD COMPLETE")
 
+    @staticmethod
+    def next_batch_generator(file):
+        while True:
+            lines = file.readlines(BATCH_FETCH_SIZE)
+            lines = list(map(lambda bianary: bianary.decode(), lines))
```

Comment:
```suggestion
            lines = list(map(lambda binary: binary.decode(), lines))
```

Reply:
map and lambda are older practices in Python. List comprehensions are often recommended.
Merging the previous line 41 and the new line would be:
`lines = [line.decode().rstrip() for line in lines]`